File: sdk/java/src/test/java/com/microsandbox/HelloWorldTest.java
package com.microsandbox;

import org.junit.Test;
import static org.junit.Assert.*;

/**
 * Test class for the HelloWorld class.
 */
public class HelloWorldTest {

    @Test
    public void testGreet() {
        String result = HelloWorld.greet("Test");
        assertTrue(result.contains("Hello, Test!"));
    }
}

File: sdk/kotlin/src/test/kotlin/com/microsandbox/HelloWorldTest.kt
package com.microsandbox

import kotlin.test.Test
import kotlin.test.assertTrue

/**
 * Test class for the HelloWorld object.
 */
class HelloWorldTest {

    @Test
    fun testGreet() {
        val result = HelloWorld.greet("Test")
        assertTrue(result.contains("Hello, Test!"))
    }
}
